client/state/data-layer/wpcom/comments/index.js

```diff
@@ -58,6 +118,15 @@ export const addComments = ( { dispatch }, { siteId, postId }, next, { comments,
 	}
 };
 
+export const writePostCommentSuccess = ( { dispatch }, { siteId, postId, parentCommentId, placeholderId }, next, comment ) => {
+	// remove placeholder from state
+	dispatch( { type: COMMENTS_REMOVE, siteId, postId, commentId: placeholderId } );
+	// add new comment to state with updated values from server
+	dispatch( { type: COMMENTS_RECEIVE, siteId, postId, comments: [ comment ], skipSort: !! parentCommentId } );
```

Comment:
if I understand you correctly you were suggesting that writing a `COMMENTS_UPDATE` action and reducer hook at this time would be too much? that's fair if so. we can further augment the comment count reducer to listen for `COMMENTS_REMOVE` actions and decrement the count…